spec/theme/plugins.spec.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+describe(\Dxw\MyTheme\Theme\Plugins::class, function () {
+    beforeEach(function () {
+        \WP_Mock::setUp();
+        \WP_Mock::wpFunction('esc_html', [
+            'return' => function ($a) {
+                return '_'.$a.'_';
+            },
+        ]);
+        $this->abspath = \org\bovigo\vfs\vfsStream::setup()->url() . '/';
```

Comment:
There's no way in PHP to mock/override the `require` function so I figured the next best thing would be to mock the file we're requiring instead.

I'm using [this package](https://github.com/mikey179/vfsStream) that creates a 'virtual' filesystem in memory.